test/testcafe/test-snippet-prod.js

```diff
@@ -1,32 +1,28 @@
-/* eslint-disable */
```

Comment:
@rhymiz This file was changed in #61, which you signed off on last year. I'm pretty sure it's just used for testing and will be fine to merge; is that correct?